asserts/sysdb/trusted.go

```diff
@@ -130,7 +130,7 @@ func init() {
 // Trusted returns a copy of the current set of trusted assertions as used by Open.
 func Trusted() []asserts.Assertion {
 	trusted := []asserts.Assertion(nil)
-	if os.Getenv("SNAPPY_USE_STAGING_STORE") != "1" {
+	if !osutil.EnvBool("SNAPPY_USE_STAGING_STORE") {
```

Comment:
(bikesheed): I wonder if there is anything similar to `EnvBool` just a bit more clear. `GetenvBool()` ? `GetBoolEnv()`? But all my idea are as readable as `EnvBool` and longer so maybe `EnvBool` is the right one :)